web_src/less/_repository.less

```diff
@@ -20,6 +20,15 @@
                 top: 5px;
             }
 
+            &.repo-title .avatar {
+                margin-left: -23px;
+                box-shadow: 0 0 0 2px #fff;
```

Comment:
Needs equivalent color in `arc-green`.

Reply:
Done via inheriting from `color`, as `box-shadow` doesn't have `-color` variant.